File: app/src/main/java/com/intelegain/agora/model/LeaveMaster.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by suraj.m on 16/8/17.
 */

public class LeaveMaster {

    @SerializedName("FiscalYear")
    @Expose
    public Integer fiscalYear;

    @SerializedName("LeaveType")
    @Expose
    private Leave leaveType;
    @SerializedName("LeaveData")
    @Expose
    private ArrayList<LeaveData> leaveData = null;

    public Leave getLeaveType() {
        return leaveType;
    }

    public void setLeaveType(Leave leaveType) {
        this.leaveType = leaveType;
    }

    public ArrayList<LeaveData> getLeaveData() {
        return leaveData;
    }

    public void setLeaveData(ArrayList<LeaveData> leaveData) {
        this.leaveData = leaveData;
    }
}

File: app/src/main/java/com/intelegain/agora/model/LeaveMaster.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

/**
 * Created by suraj.m on 16/8/17.
 */
class LeaveMaster {
    @kotlin.jvm.JvmField
    @SerializedName("FiscalYear")
    @Expose
    var fiscalYear: Int? = null
    @SerializedName("LeaveType")
    @Expose
    var leaveType: Leave? = null
    @SerializedName("LeaveData")
    @Expose
    var leaveData: ArrayList<LeaveData>? = null

}